less/index.less

```diff
@@ -57,3 +57,102 @@ html, body {
     height: 100%;
 }
 
+// end of original CSS code
+
+//set font size to 62.5% to allow easy use of 10-based rems
+html {
+  font-size: 62.5%;
+}
+
+//set base font size to 16px
+body {
+  font-size: 1.6rem;
+}
+
+// VAR VAR VAR
+@splash: #E9ECEF;
+@mainfont: 'Merriweather', serif;
+
+.main-font() {
+  font-family: @mainfont
+}
+
+//define flex() mixin
+.flex(@justify) {
+  display: flex;
+  justify-content: @justify;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+.container {
+  .main-font;
+  max-width: 900px;
+  margin: 0 auto;
+  background: white;
+  padding: 0 4%;
+  header {
+    padding: 20px 0;
+    .flex(flex-end);
+  }
+  nav {
+    width: 50%;
+    .flex(flex-end);
+    a {
+      margin-left: 10%;
+    }
+  }
+  .splash {
+    background: @splash;
+    width: 100%;
+    padding: 50px 5% 25px 5%;
+    .flex(flex-start);
+    flex-direction: column;
+    align-items: flex-start;
+    img {
+      width: 450px;
+      max-width: 100%;
+    }
+    h1 {
+      margin-top: 30px;
+      margin-bottom: 20px;
+      font-size: 3rem;
+    }
+    p.intro {
+      margin-bottom: 30px;
+      line-height: 1.25;
+    }
+    h2 {
+      margin-bottom: 30px;
+      font-size: 2.4rem;
+    }
+    li {
+      list-style-type: disc;
+      list-style-position: inside;
+      line-height: 1.5;
+    }
+    .skills {
+      line-height: normal;
+      width: 100%;
+      columns: 3;
+      margin-bottom: 40px;
+      @media (max-width: 800px) {
```

Comment:
good job with successfully nesting the media queries. You clearer know what's required and executed on it correctly